workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/README.md

```diff
@@ -0,0 +1,5 @@
+This project exists in folder prefixed with a '.' so that Unity ignores it at script compile time, but it can still be
+distributed as part of the Unity package.
+
+Unity doesn't add a project reference to `System.IO.Compression.FileSystem`, which is required for access to the `ZipFile` class.
```

Comment:
I don't believe this project uses `ZipFile`, so can you walk us through why this is best done as a separate project and binary?

Reply:
Oh, this README.md has not been updated. I copied it over from the .DownloadCoreSdk project.

The reason this is a separate project is that it uses the Platform SDK, which requires NuGet to be able to download it without going insane, as Unity lacks NuGet support (due to the transitive dependencies on gRPC and prerequisite libraries).